src/pages/404.js

```diff
@@ -0,0 +1,13 @@
+import React from "react"
+import {Link} from "gatsby"
+
+const KrakenPage = () => {
```

Comment:
**nitpick:** This one should be named `NotFoundPage`

The reason it's important to name things what they are is if someone comes in and gets an error it should make sense to them as well.

Reply:
That's ok.
😺 
look at the [comment above](https://github.com/raae/pow-app/pull/269#discussion_r629935885)

I just needed to make a drawing after the coding, you know:

![Batman_on_the_beach](https://user-images.githubusercontent.com/7642493/118241931-04e65080-b49d-11eb-88bc-d7fff11759c3.jpeg)

[Photo](https://unsplash.com/photos/0xOCVe7nUU0) by [Zakaria Ahada](https://unsplash.com/@zakariahada) on [Unsplash](https://unsplash.com/@zakariahada).

Reply:
I liked the one above the best, it looks a little lost.